frappe/integrations/doctype/ldap_settings/ldap_settings.py

```diff
@@ -21,17 +21,17 @@ def validate_ldap_credentails(self):
 		except ImportError:
 			msg = """
 				<div>
-					Seems ldap is not installed on system.<br>
-					Guidelines to install ldap dependancies and python package,
-					<a href="https://discuss.erpnext.com/t/frappe-v-7-1-beta-ldap-dependancies/15841" target="_blank">Click here</a>,
+					{{_(Seems ldap is not installed on system.<br>
```

Comment:
`{{_("Seems ldap is not installed on system.<br>...")}}`

@almeidapaulopt please add the quotes to string